stellarphot/core.py

```diff
@@ -62,3 +64,307 @@ def __init__(self, gain=1.0 * u.electron / u.adu,
         self.gain = gain
         self.read_noise = read_noise
         self.dark_current = dark_current
+
+
+class BaseEnhancedTable:
+    """
+    A base class to hold an `astropy.table.Table` data table and extend it
+    so that chosen columns can be accessed as attributes.  This base class
+    imposes the requirement that the table have columns corresponding to
+    attributes 'id', 'ra', and 'dec' attributes, as well as confirming
+    the 'ra' and 'dec' are in degrees.
+
+    Parameters
+    ----------
+
+    table_descript: `numpy.ndarray`
```

Comment:
```suggestion
    table_description: `numpy.ndarray`
```

May as well complete the word...

Reply:
OK, done throughout code.